build/testing/config-browsers.js

```diff
@@ -34,293 +202,56 @@ exports.Phases = Phases;
-                    }
+                    Windows10_MicrosoftEdge,
+                    Windows10_IE,
+                    Windows10_Chrome_68,
+                    Windows10_Firefox,
+                    MacOS10_Safari_11_1,
+                    IOS_IPhoneX_Safari_Portrait,
+                    Android_6_0_Chrome_Portrait
                 ];
-            
-            case Phases.beta:
-                
+            case Phase.beta:
                 return [
-                    {
-                        'platform': 'Windows 10',
-                        'browserName': 'MicrosoftEdge',
-                        'version': '17.17134',
-                        'screenResolution': '1280x1024'
-                    },
-                    {
-                        'platform': 'Windows 10',
-                        'browserName': 'internet explorer',
-                        'version': '11.103',
-                        'screenResolution': '1400x1050'
-                    },
-                    {
-                        'platform': 'Windows 10',
-                        'browserName': 'chrome',
-                        'version': '68.0',
-                        'screenResolution': '1920x1200'
-                    },
-                    {
-                        'platform': 'Windows 10',
-                        'browserName': 'firefox',
-                        'version': '61.0',
-                        'screenResolution': '2560x1600'
-                    },
-                    {
-                        'platform': 'macOS 10.13',
-                        'browserName': 'safari',
-                        'version': '11.1',
-                        'screenResolution': '2048x1536'
-                    },
-                    {
-                        'platformName': 'iOS',
-                        'platformVersion': '11.3',
-                        'deviceOrientation': 'PORTRAIT',
-                        'deviceName': 'iPhone X Simulator',
-                        'appiumVersion': '1.8.1',
-                        'browserName': 'Safari'
-                    },
-                    {
-                        'platformName': 'iOS',
-                        'platformVersion': '9.3',
-                        'deviceOrientation': 'PORTRAIT',
-                        'deviceName': 'iPad Pro Simulator',
-                        'appiumVersion': '1.7.1',
-                        'browserName': 'Safari'
-                    },
-                    {
-                        'platformName': 'Android',
-                        'platformVersion': '6.0',
-                        'deviceOrientation': 'PORTRAIT',
-                        'deviceName': 'Android Emulator',
-                        'appiumVersion': '1.8.1',
-                        'browserName': 'Chrome',
-                        'language': 'en',
-                        'locale': 'en_US'
-                    }
+                    Windows10_MicrosoftEdge,

```

Comment:
I don't see the advantages of this set of changes on the configuration.  These configurations will change quarterly and are provided by platform configurators of the service.  Your solution feels less maintainable and certainly isn't as friendly to read.  Now if I go into the phases to see what's being tested I have to find the const, look at it's properties, and return back to the switch.  It's better in my opinion to keep it as it is.

Reply:
What maybe better is to just create three separate configuration files which get imported into a const object that can hold the entire collection.  I can see where you eliminated some duplication by using constants, but, I think in this case duplication is ok given how this is used.  Otherwise, your const name should be generated by every property of the array or details get lost.  Like resolution, versions, etc.

Reply:
There is a lot of duplication between the alpha/beta/releases, that's why. This helps solve that, now if you want to update, you can just remove the named config and the variable assigned to that config. Otherwise it's very messy to try and find the right items. Opening this up for comment thought, @chrisdholt @nicholasrice thoughts?

Reply:
That's true, why three configurations? If its one per phase then we have the same duplication problem. I think it might be better to go with your suggestion on the naming convention of the variable to make it more precise.

Reply:
If it's duplicated, then abstract it - that's the rule I try to follow. We could also implement a "common" array if they are shared across all phases.

Reply:
To that end, if these configurations are provided by a service, can we not pull from that service?

Reply:
I think it's unique in the way it's used.  When building out the matrix you want to see everything together because it's easier to make sure you have the right combination of all the different properties.

One idea may be to separate configuration files.  config-browsers.alpha.js which has min requirements, then config-browsers.beta.js which imports the alpha config and appends to the beta config.  Then the config-browsers.release.js which imports beta config and appends the release ones.  This way it's not duplicate, still maintainable, and I think might check the collective boxes we're trying to achieve?